addon/register-units.js

```diff
@@ -93,3 +94,15 @@ registerUnit(['psi'], psi);
 
 let atmosphere = Unit.scale(101325, pascals);
 registerUnit(['atmosphere', 'at'], atmosphere);
+
+// Volume
+
+let liters = Unit.root();
+registerUnit(['litres', 'liters', 'L', 'l'], liters);
```

Comment:
Swap the litres line with the millilitres lines so that the units read smallest to largest.